### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by altontonn**
### Contact Form

-  Created form validation to prompt the user to enter all credentials before submitting.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,10 +1,250 @@
...
+    const useremail = userEmail.value.trim().toLowerCase()
+    let valid = false
+    const pattern = /[A-Z]/
+    if(pattern.test(userEmail.value)) {
+      showError(userEmail, 'Email has to be in lowercase')
+    } else if (!required(useremail)) {
+      showError(userEmail, 'Please enter email')
+    } else if (!emailValidation(useremail)) {
+      showError(userEmail, 'Please enter a valid email')
+    }
```
Comment (OLIPLICHE) [inline js/main.js:196]: - [ ] Please If a user submits the contact form with the email field containing uppercase characters an error message should show up, and The error message should show up close to the "submit" button.
- This is from your Project
-
a
- Kindly fix that and submit again for review

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by altontonn** [comment]
Hey  there what do you mean error message near button?

**Review by Ginohmk — APPROVED** [review]
Hi @altontonn  

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**